```
pkg/aws/ocpgwdeployer.go:187:10: error returned from external package is unwrapped: sig: func errors.Join(errs ...error) error (wrapcheck)
		return goerrors.Join(errs...)
		       ^
pkg/aws/ocpgwdeployer.go:220:9: error returned from external package is unwrapped: sig: func errors.Join(errs ...error) error (wrapcheck)
	return goerrors.Join(errs...)
	       ^
pkg/aws/ocpgwdeployer.go:446:9: error returned from external package is unwrapped: sig: func errors.Join(errs ...error) error (wrapcheck)
	return goerrors.Join(errs...)
	       ^
```
[Go](https://github.com/submariner-io/cloud-prepare/actions/runs/15308426738/job/43066893180?pr=1143#logs)

--- a/.golangci.yml
+++ b/.golangci.yml
@@ -59,16 +59,8 @@ linters:
         # Allow unnecessary embedded field selectors
         - -QF1008
     wrapcheck:
-      ignore-sigs:
+      extra-ignore-sigs:
         - .Error(
-        - .Errorf(
-        - errors.New(
-        - errors.Unwrap(
-        - .Wrap(
-        - .Wrapf(
-        - .WithMessage(
-        - .WithMessagef(
-        - .WithStack(
     wsl:
       allow-cuddle-declarations: true
   enable:

--- a/pkg/aws/ocpgwdeployer.go
+++ b/pkg/aws/ocpgwdeployer.go
@@ -21,6 +21,7 @@ package aws
 import (
 	"bytes"
 	"context"
+	goerrors "errors"
 	"fmt"
 	"text/template"
 
@@ -32,7 +33,6 @@ import (
 	"github.com/submariner-io/cloud-prepare/pkg/ocp"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime/serializer/yaml"
-	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 )
 
 type ocpGatewayDeployer struct {
@@ -184,7 +184,7 @@ func (d *ocpGatewayDeployer) validateDeployPrerequisites(vpcID string, input api
 	errs = appendIfError(errs, err)
 
 	if err != nil {
-		return utilerrors.NewAggregate(errs)
+		return goerrors.Join(errs...)
 	}
 
 	// If instanceType is not specified, auto-select the most suitable one.
@@ -217,7 +217,7 @@ func (d *ocpGatewayDeployer) validateDeployPrerequisites(vpcID string, input api
 		errs = appendIfError(errs, d.aws.validateCreateTag(*subnets[0].SubnetId))
 	}
 
-	return utilerrors.NewAggregate(errs)
+	return goerrors.Join(errs...)
 }
 
 type machineSetConfig struct {
@@ -443,7 +443,7 @@ func (d *ocpGatewayDeployer) validateCleanupPrerequisites(vpcID string) error {
 		errs = appendIfError(errs, d.aws.validateRemoveTag(subnets[0].SubnetId))
 	}
 
-	return utilerrors.NewAggregate(errs)
+	return goerrors.Join(errs...)
 }
 
 func (d *ocpGatewayDeployer) deleteGateway(publicSubnet *types.Subnet) error {

